crates/orchestrator/src/tests/common/mod.rs

```diff
@@ -74,3 +77,12 @@ pub fn custom_job_item(default_job_item: JobItem, #[default(String::from("0"))]
 
     job_item
 }
+
+pub async fn drop_database() -> color_eyre::Result<()> {
```

Comment:
Your ideology is valid, even we wanted to do the same but the `Drop` trait desn't support `async` function implementation, hence we went with this implementation.






### IMPORTANT 
[This branch is a fork from [increasing coverage](https://github.com/madara-alliance/madara-orchestrator/tree/feat/increasing-test-coverage) ]
[This comment refers to changed already merged to main, they are shown here since #52 is merged and the base of this PR automatically shifted to `main` ]